### DISCUSSION TITLE
Truenas Scale stuck on Deploying

### DISCUSSION THREAD

**Discussion by glenmorangie007** [guydavis/machinaris#523] [General]
Can anyone help - I have followed the instructions to install Machinaris on my TrueNas Scale but it is stuck on Deploying and will not go active

using : v0.6.1_1.0.11

Any Advice please

Thanks

**Comment by guydavis** [comment]
Apologies, the [[TrueNAS](https://github.com/guydavis/machinaris/wiki/TrueNAS) support in Machinaris was contributed by a TrueNAS user.  Your best option is to post your question in the #truenas channel of the [Machinaris Discord](https://discord.gg/mX4AtMTt87). Lots of helpful TrueNAS users there.
